guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -580,7 +580,7 @@ following:
 $ git fetch upstream
 $ git checkout my_pull_request
 $ git rebase upstream/master
-$ git rebase -i
+$ git rebase -i <after-this-commit-from-master>
```

Comment:
This is even better thank you! I am making a change as:
    
    git rebase -i upstream/master